pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf

```diff
@@ -1658,9 +1658,9 @@
     }
     "questions": {
       "send_records_options": {
-        "help_me_get_via_portal": "I would like the Project Singular team to assist me by phone or Zoom call, walking me through obtaining the necessary records via my online patient portal. I understand that someone from Boston Children’s Hospital will reach out to me to schedule a time that is convenient for me to meet with them by phone or Zoom.",
-        "upload": "I will upload the records myself via the Medical Record Upload button on my Project Singular Dashboard. This will include my most recent cardiology clinic note, echo report, catheterization report, and operation notes."
-        "fax": "I will fax or email them to the team at Boston Children’s Hospital, fax to (617) 730-6201 or email to <a href=\"mailto:contact@projectsingular.org\" class=\"Link\">contact@projectsingular.org</a>. This will include my most recent cardiology clinic note, echo report, catheterization report, and operation notes."
+        "help_me_get_via_portal": "I would like the Project Singular team to assist me by phone or Zoom call, walking me through obtaining the necessary records via $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"my child's\", \"my\") online patient portal. I understand that someone from Boston Children’s Hospital will reach out to me to schedule a time that is convenient for me to meet with them by phone or Zoom.",
+        "upload": "I will upload the records myself via the Medical Record Upload button on $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"my child's\", \"my\") Project Singular Dashboard. This will include $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"my child's\", \"my\") most recent cardiology clinic note, echo report, catheterization report, and operation notes."
+        "fax": "I will fax or email them to the team at Boston Children’s Hospital, fax to (617) 730-6201 or email to <a href=\"mailto:project_singular@childrens.harvard.edu\" class=\"Link\">project_singular@childrens.harvard.edu</a>. This will include $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"my child's\", \"my\") most recent cardiology clinic note, echo report, catheterization report, and operation notes."
       },
```

Comment:
The email we had was wrong. Referred back to document with ticket. The correct address here is this childrens.harvard.edu address.